build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -124,14 +124,8 @@ spec:
       - args:
         - --config
         - /etc/flow-aggregator/flow-aggregator.conf
-        - --logtostderr=false
-        - --log_dir=/var/log/flowaggregator
-        - --alsologtostderr
-        - --log_file_max_size=100
-        - --log_file_max_num=4
+        - --logtostderr
         - --v=0
```

Comment:
This change may be orthogonal to this PR. What's your take on this @srikartati?
It's fine to keep logging to `/var/log/flowaggregator` as long we use a hostPath volume, unless there is a bug in klog an it tries to log to `/tmp` even when `--log_dir` is provided (which I would find surprising).

Reply:
Yes, I think this change seems to be orthogonal. 

https://github.com/vmware-tanzu/antrea/blob/main/cmd/flow-aggregator/main.go#L51
We expect log limits here. Not sure what default values or how does this function will react. @dreamtalen do you know what happens?

Reply:
I quickly tried this patch by adding the missing hostPath volume. I did not get any error that logs are written to /tmp.

Here is my diff on top of this patch:
```
~/work/antrea_all/antrea$ git diff
diff --git a/build/yamls/flow-aggregator/base/flow-aggregator.yml b/build/yamls/flow-aggregator/base/flow-aggregator.yml
index 5dd33d93..7e7ac18d 100644
--- a/build/yamls/flow-aggregator/base/flow-aggregator.yml
+++ b/build/yamls/flow-aggregator/base/flow-aggregator.yml
@@ -124,7 +124,11 @@ spec:
       - args:
         - --config
         - /etc/flow-aggregator/flow-aggregator.conf
-        - --logtostderr
+        - --logtostderr=false
+        - --log_dir=/var/log/flow-aggregator
+        - --alsologtostderr
+        - --log_file_max_size=100
+        - --log_file_max_num=4
         - --v=0
         name: flow-aggregator
         image: flow-aggregator
@@ -135,9 +139,14 @@ spec:
           name: flow-aggregator-config
           readOnly: true
           subPath: flow-aggregator.conf
+        - mountPath: /var/log/flow-aggregator
+          name: host-var-log-flow-aggregator
       serviceAccountName: flow-aggregator
       volumes:
       - name: flow-aggregator-config
         configMap:
           name: flow-aggregator-configmap
-        
+      - name: host-var-log-flow-aggregator
+        hostPath:
+          path: /var/log/flow-aggregator
+          type: DirectoryOrCreate
```

Reply:
Thanks for checking let's go with this

Reply:
I think keeping the same behavior as antrea-controller for logging is better.
@antoninbas Do you know what is the benefit of logging this to a separate directory along with stderr? Where will the directory be used? I see that log collection in e2e tests is also using "kubectl logs.."

Reply:
The idea was to have all the logs in one directory on the Node (`/var/log/antrea`). But it doesn't make too much sense for the Antrea Controller and the Flow Aggregator which are Deployments, and not DaemonSets. You can either match the Antrea Controller and we can revisit this decision later for both, or you can switch to only logging to stdout / stderr (remove file logging) for the Flow Aggregator (leave the Antrea Controller alone in this PR though). I don't have a strong preference. Maybe the second solution simplifies things and is the right way to go? Since we were not actually logging to the Node filesystem earlier (missing volume mount) there isn't any loss of functionality. What do you think?

Reply:
Thanks, Antonin.
From [Jianjun's PR](https://github.com/vmware-tanzu/antrea/pull/562#issuecomment-605688375), I see that it can be used for the log bundle. If that is the case, can we use `/var/log/antrea` instead of `/var/log/flow-aggregator` here as a solution? Earlier, the missing log functionality in the flow aggregator is overlooked. We can fix that here.

Reply:
yes I think you can use `/var/log/antrea`

Reply:
Thanks Srikar for helping me try adding the hostpath, updated PR with `/var/log/antrea` log path.